content/en/agent/proxy.md

```diff
@@ -316,7 +365,8 @@ backend datadog-logs
 
 The file might be located at `/etc/ssl/certs/ca-bundle.crt` for CentOS, Redhat.
 
-Once the HAProxy configuration is in place, you can reload it or restart HAProxy. **It is recommended to have a `cron` job that reloads HAProxy every 10 minutes** (usually doing something like `service haproxy reload`) to force a refresh of HAProxy's DNS cache, in case `app.datadoghq.eu` fails over to another IP.
+HAProxy 1.8 and newer allows DNS service discovery to detect server changes and automatically apply them to configuration.
+If you are using older version of HAProxy, you have to reload or restart HAProxy. **It is recommended to have a `cron` job that reloads HAProxy every 10 minutes** (usually doing something like `service haproxy reload`) to force a refresh of HAProxy's DNS cache, in case `app.datadoghq.eu` fails over to another IP.
```

Comment:
Is there a doc from HAproxy we could link to for this Best practice?

Reply:
I haven't come across any best practices document but, for reference, I can recommend the following link - 
https://www.haproxy.com/blog/dns-service-discovery-haproxy/
https://cbonte.github.io/haproxy-dconv/1.8/configuration.html#5.3